src/main/java/am/ik/yavi/constraint/CharSequenceConstraint.java

```diff
@@ -134,6 +118,29 @@ public CharSequenceConstraint(Normalizer.Form normalizerForm,
 		return this;
 	}
 
+	/**
+	 * Does the given value start with the {@code prefix}
+	 * @param prefix the prefix the value has to start with
+	 */
+	public CharSequenceConstraint<T, E> startsWith(CharSequence prefix) {
```

Comment:
Does the argument type need to be `CharSequence`? Isn't `String` enough?

Reply:
The main benefit of using `CharSequence` over `String` is that we can use not only `String` but also classes such as `StringBuilder` and so on.

I also added few test cases that also cover this.